### PR TITLE
Release 9.5.0

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** PR for release 9.5.0 from main branch at 1f1c17379d8c44143b9a3a735b1065e140867ea6

Addresses [CUMULUS-1: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

## Changes

* Release 9.5.0

### CODE REVIEW

File: example/cumulus-tf/main.tf [hunk 1]
```diff
@@ -66,6 +66,8 @@ data "aws_ecr_repository" "async_operation" {
 
 module "cumulus" {
   source = "../../tf-modules/cumulus"
+
+  lambda_timeouts = var.lambda_timeouts
```
Comment (npauzenga) [inline example/cumulus-tf/main.tf]: Where did this come from? Might be right but I'm surprised to see something other than version bumps and docs.

**Review by npauzenga — APPROVED** [review]
Looks good @Jkovarik. Just making sure, there isn't a dashboard release that depends on this? Seems like no.

File: example/cumulus-tf/main.tf (hunk 1 above)
Comment (npauzenga) [reply example/cumulus-tf/main.tf]: Nevermind, looks like you removed it.

Comment (Jkovarik) [reply example/cumulus-tf/main.tf]: Right you are, I pushed up a change that removed it, sorry about that.

**Pull request merged**